src/orangeClockFunctions/displayBlockMoscowFees.py

```diff
@@ -70,6 +70,14 @@ def getMoscowTime():
     return moscowTime
 
 
+def getFiatPrice(currency):
+    gc.collect()
+    data = urequests.get("https://mempool.space/api/v1/prices")
+    price = data.json()[currency]
+    fiatPrice = f"{price:,}"

```

Comment:
This repetition of code from "getPrice" violates DRY (Don't Repeat Yourself).

Simpler to just do:
```python
def getPriceDisplay(currency):
    return f"{getPrice(currency):,}"
```

Could also add a nice-to-have nod to the Europeans (I think they all swap comma and period separators):
```python
    ...
    price_str = f"{getPrice(currency):,}"
    if currency == "EUR":
        price_str = price_str.replace(",", ".")
    return price_str
```

Reply:
Great feature to swap , with . for the EUR